tests/test_vxlan_vtep_vni.rb

```diff
@@ -210,4 +214,23 @@ def test_suppress_arp
     vni.suppress_arp = vni.default_suppress_arp
     refute(vni.suppress_arp, 'suppress_arp should be disabled')
   end
+
+  def test_suppress_uuc
+    skip('Platform does not support suppress-unknown-unicast') unless
+    node.cmd_ref.supports?('vxlan_vtep_vni', 'suppress_uuc')
```

Comment:
Well - I'm not getting an "UnsupportedError exception" from a platform that should be unsupported.